app/index.html

```diff
@@ -181,18 +181,20 @@
                    class='btn btn-sm btn-default'>
           </form>
 
-          <input type="checkbox"
-                 name="checked_files[]"
-                 value="Select all"
-                 ng-checked="all_selected"
-                 ng-click="select_all()">
-          Select all
-          <table>
+          <table class='table table-striped table-hover table-condensed'>
           <tr>
+            <th>
+              <input type="checkbox"
+                     name="checked_files[]"
+                     value="Select all"
+                     ng-checked="all_selected"
+                     ng-click="select_all()">
+            </th>
             <th>Filename</th>
             <th>Size</th>
             <th>Last modified</th>
             <th>Tags</th>
+            <th><i class="fa fa-minus-square" ng-click="clear()"></i></th>
```

Comment:
Could also do `file_list.clear()` and not need to bind a new object into the scope?